### DISCUSSION TITLE
Possible to get existing video thumbnails recreated with ffmpeg?

### DISCUSSION THREAD

**Discussion by hess8** [LycheeOrg/Lychee#2023] [Q&A]
On my new server I got video thumbnails working with ffmpeg (very nice).  On my existing albums is there a way to force it recreate the video thumnails, without deleting and reloading them?  Right now those have no thumbnail, just the play button.  

Thanks

**Comment by d7415** [comment]
Try `php artisan lychee:generate_thumbs`
